liquibase-core/src/test/java/com/example/liquibase/parser/java/ExampleJavaChangelog.java

```diff
@@ -0,0 +1,55 @@
+package com.example.liquibase.parser.java;
+
+import liquibase.change.core.RenameTableChange;
+import liquibase.parser.core.java.ChangeSet;
+import liquibase.precondition.core.PreconditionContainer;
+import liquibase.precondition.core.TableExistsPrecondition;
+
+public class ExampleJavaChangelog {
+
+    @ChangeSet(id = "1", author = "nvoxland", order = 1)
+    public void firstChange() {
```

Comment:
I don't have this method doing anything and don't have any parameters so far. We'd need logic in the JavaStatement class to look at parameter types and inject value we'd know. Scope should maybe be all we need?